pandas/tools/util.py

```diff
@@ -74,6 +110,8 @@ def to_numeric(arg, errors='raise'):
     >>> import pandas as pd
     >>> s = pd.Series(['1.0', '2', -3])
     >>> pd.to_numeric(s)
+    >>> pd.to_numeric(s, downcast=np.int8)
```

Comment:
add using ``pd.to_numeric(s, downcast=signed)`` 

I thin you also should show the results of these yes?